napari/_qt/_qapp_model/_tests/test_qapp_model_menus.py

```diff
@@ -2,27 +2,22 @@
 
 import pytest
 
-from napari import viewer
 from napari._app_model import constants, get_app
 from napari._qt._qapp_model import build_qmodel_menu
-from napari._qt._qapp_model.qactions import init_qactions
-from napari._qt.qt_main_window import Window
 
 
+# `builtins` required so there are samples registered, so samples menu exists
 @pytest.mark.parametrize('menu_id', list(constants.MenuId))
-def test_build_qmodel_menu(qtbot, menu_id):
-    """Test that we can build qmenus for all registered menu IDs"""
+def test_build_qmodel_menu(builtins, make_napari_viewer, qtbot, menu_id):
+    """Test that we can build qmenus for all registered menu IDs."""
     app = get_app()
 
-    mock = MagicMock()
-    with app.injection_store.register(
-        providers={viewer.Viewer: lambda: mock, Window: lambda: mock}
-    ):
```

Comment:
I don't think injection is needed because we register providers in `init_qactions`, similar to here: https://github.com/napari/napari/pull/4977/files#r1013731145